apps/packages/takes_package_service.py

```diff
@@ -40,7 +40,7 @@ def get_take_package_id(self, item):
         :return: _id of the package or None
         """
         takes_package = [package.get(PACKAGE) for package in item.get(LINKED_IN_PACKAGES, [])
-                         if package.get(PACKAGE_TYPE)]
+                         if package.get(PACKAGE_TYPE) is TAKES_PACKAGE]
```

Comment:
With the above change, text items are getting delivered to `Digital Subscribers`. Hence you had to change the number in the test. 

The change should be `if package.get(PACKAGE_TYPE) == TAKES_PACKAGE`. I want to see what the data looks like. Can you export the document from mongo?